client/src/components/formHelpers/form-validators.tsx

```diff
@@ -9,6 +9,18 @@ const fCCRegex =
 const localhostRegex = /localhost:/;
 const httpRegex = /http(?!s|([^s]+?localhost))/;
 
+function hasEndpoint(urlString: string): boolean {
+  try {
+    const url = new URL(urlString);
+    if (url.pathname && url.pathname !== '/') {
+      return true;
+    }
+  } catch (_e) {
+    return false;
+  }
+  return false;
```

Comment:
```suggestion
  try {
    return new URL(urlString).pathname !== '/';
  } catch {
    return false;
  }
```
`if (condition) { return true } else { return false }` (which this boiled down to) is a bit of an anti-pattern.

Reply:
🤫